app/models/resource.ts

```diff
@@ -0,0 +1,29 @@
+import { AsyncBelongsTo, attr, belongsTo } from '@ember-data/model';
+import RegistrationModel from 'ember-osf-web/models/registration';
+
+import OsfModel from './osf-model';
+
+export enum resourceTypes {
```

Comment:
I think other enums we have are in PascalCase
```suggestion
export enum ResourceTypes {
```